webservices/templates/swagger-ui.html

```diff
@@ -30,14 +29,85 @@
 </head>
 
 <body>
-    <div id="swagger-ui"></div>
+
+<div id="swagger-ui"></div>
+ <script type="text/javascript">
+   /* * * CONFIGURATION VARIABLES: EDIT BEFORE PASTING INTO YOUR WEBPAGE * * */
+   var apiUmbrellaSignupOptions = {
+     // Pick a short, unique name to identify your site, like 'gsa-auctions'
+     // in this example.
+     registrationSource: 'gsa-auctions',
+
+     // Enter the API key you signed up for and specially configured for this
+     // API key signup embed form.
+     apiKey: "{{ api_key }}",
+     // document.write(apiKey.toLocaleString( ));
+     // Provide a URL or e-mail address to be used for customer support.
+     //
+     // The format for e-mail addresses can be given as either
+     // 'example@example.com' or 'mailto:example@example.com'.
+     contactUrl: 'https://agency.gov/contact',
```

Comment:
Thanks for your work on this, @jason-upchurch! Can you update some of these configuration variables? `contactUrl`, `siteName`, `emailFromName`, `exampleApiUrl`, `verifyEmail` and `sendWelcomeEmail` look like the essential ones to me.